examples/blueprint.yaml

```diff
@@ -0,0 +1,124 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.5/types.yaml
+  # For use with a manager use: plugin:cloudify-openstacksdk-plugin
+  - plugin.yaml
+
+inputs:
+
+  example_subnet_cidr:
+    type: string
+    default: 10.10.0.0/24
+
+  auth_url:
+    type: string
+
+  username:
+    type: string
+
+  password:
+    type: string
+
+  region_name:
+    type: string
+
+  project_name:
+    type: string
+
+dsl_definitions:
+
+  client_config: &client_config
+    auth_url: { get_input: auth_url }
+    username: { get_input: username }
+    password: { get_input: password }
+    region_name: { get_input: region_name }
+    project_name: { get_input: project_name }
+
+node_templates:
+
+  example_router:
+    type: cloudify.nodes.openstack.Router
+    properties:
+      client_config: *client_config
+      resource_config:
+        name: 'example_router'
+
+  example_network:
+    type: cloudify.nodes.openstack.Network
+    properties:
+      client_config: *client_config
+      resource_config:
+        name: 'example_network'
+
+  example_security_group:
+    type: cloudify.nodes.openstack.SecurityGroup
+    properties:
+      client_config: *client_config
+      resource_config:
+        name: 'example_security_group'
+        description: 'A security group created by Cloudify OpenStack SDK plugin.'
+
+  example_subnet:
+    type: cloudify.nodes.openstack.Subnet
+    properties:
+      client_config: *client_config
+      resource_config:
+        name: 'example_subnet'
+        network_id: { get_attribute: [ example_network, resource_id ] }
+        cidr: { get_input: example_subnet_cidr }
+        enable_dhcp: true
+        ip_version: 4
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: example_network
+    # This is only run on local mode. For use with a manager, it can be commented out.
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          inputs:
+            resource_config: { get_property: [ SELF, resource_config ] }
```

Comment:
@EarthmanT Do we still need to pass the resource_config here ? since each resource on the task module only accepts the `openstack_resource` and it seems it me no need to pass here if we already handle them inside the wrapper ?

Reply:
It is because I am running this blueprint with cfy local profile and `get_attribute` is not supported with local profile.

Reply:
I'm not sure how this related to passing `resource_config` to the create task ? because right now we are not supporting and allow `resource_config` to be passed to the task, check this 

https://github.com/cloudify-incubator/cloudify-openstacksdk-plugin/blob/trammell-changeset-dev2/openstacksdk_plugin/resources/network/subnet.py#L24